pkgs/desktops/gnome-3/3.20/core/gnome-shell/default.nix

```diff
@@ -15,7 +16,8 @@ stdenv.mkDerivation rec {
 
   buildInputs = with gnome3;
     [ gsettings_desktop_schemas gnome_keyring gnome-menus glib gcr json_glib accountsservice
-      libcroco intltool libsecret pkgconfig python3 libsoup polkit libcanberra gdk_pixbuf librsvg
+      libcroco intltool libsecret pkgconfig python3 python3Packages.pygobject3
+      libsoup polkit libcanberra gdk_pixbuf librsvg
       clutter networkmanager libstartup_notification telepathy_glib docbook_xsl docbook_xsl_ns
```

Comment:
what is the gain of this? How does it work if this derivation (`gnome-shell`) depends on a derivation that itself has python dependencies?